### PR TITLE
Update schema.graphql

### PR DESCRIPTION
**Pull request by PaulieB14**
Towards https://github.com/RequestNetwork/requestNetwork/issues/1017

Added descriptions to Schema in preparation to add docs to your docs page

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -56,15 +88,26 @@ enum EventName {
 }
 
 type EscrowEvent @entity {
+  "date payment set up"
   id: ID!
+  "network contract address"
   contractAddress: Bytes!
+  "is the reference data used to track the escrow event"
   reference: Bytes!
+  "smart contract that triggers when a payment is dispursed"
```
Comment (MantisClone) [inline schema.graphql:97]: Minor spelling error. dispersed should be disbursed. https://www.vocabulary.com/articles/chooseyourwords/disperse-disburse/#:~:text=disperse%2F%20disburse,want%20your%20money%20to%20disperse!

File: schema.graphql [hunk 2]
```diff
@@ -1,25 +1,43 @@
...
   contractAddress: Bytes!
+  "contract of token used for payment"
   tokenAddress: Bytes
+  "address payment was made from"
   to: Bytes!
+  "address payment was made to"
   from: Bytes!
+  "is the reference data used to track the transfer"
   reference: Bytes!
+  "block payment took place"
```
Comment (MantisClone) [inline schema.graphql:14]: Recommend changing to "block in which payment took place"

File: schema.graphql [hunk 3]
```diff
@@ -1,25 +1,43 @@
 type Payment @entity {
+  "hash transaction of the payment"
   id: ID!
+  "network contract address"
   contractAddress: Bytes!
+  "contract of token used for payment"
   tokenAddress: Bytes
+  "address payment was made from"
```
Comment (MantisClone) [inline schema.graphql:8]: from should be to

File: schema.graphql [hunk 4]
```diff
@@ -1,25 +1,43 @@
...
   to: Bytes!
+  "address payment was made to"
```
Comment (MantisClone) [inline schema.graphql:10]: to should be from

File: schema.graphql [hunk 5]
```diff
@@ -1,25 +1,43 @@
 type Payment @entity {
+  "hash transaction of the payment"
   id: ID!
+  "network contract address"
```
Comment (MantisClone) [inline schema.graphql:4]: Please change to "payment network contract address"

File: schema.graphql [hunk 6]
```diff
@@ -1,25 +1,43 @@
...
   gasUsed: BigInt!
+  "gas fee at time of payment"
   gasPrice: BigInt!
+  "amount of payment"
   amount: BigDecimal!
 
   # fee values
+  "fee charged for payment"
   feeAmount: BigDecimal
+  "address of person paying fee"
```
Comment (MantisClone) [inline schema.graphql:30]: I suspect this is incorrect and should be the "address where the fee is sent", but I need to confirm

File: schema.graphql [hunk 7]
```diff
@@ -31,19 +49,33 @@ enum EscrowState {
 }
 
 type Escrow @entity {
+  "hash transaction of the escrow"
   id: ID!
+  "network contract address"
   contractAddress: Bytes!
+  "identifies a recipient and their payment account"
```
Comment (MantisClone) [inline schema.graphql:56]: I'm not sure this is correct. I'll have to double check.

File: schema.graphql [hunk 8]
```diff
@@ -31,19 +49,33 @@ enum EscrowState {
...
   creationTimestamp: Int!
+  "state of the escrow"
   escrowState: EscrowState!
+  "contract of token used for payment"
   tokenAddress: Bytes!
+  "amount of escrow set up"
   amount: BigDecimal!
+  "fee charged for escrow"
   feeAmount: BigDecimal!
+  "address of person paying escrow"
```
Comment (MantisClone) [inline schema.graphql:72]: I suspect this is the address where fees are sent, but I need to double check.

File: schema.graphql [hunk 9]
```diff
@@ -56,15 +88,26 @@ enum EventName {
 }
 
 type EscrowEvent @entity {
+  "date payment set up"
```
Comment (MantisClone) [inline schema.graphql:91]: Is this true? The ID is the date? I'll have to double check.

File: schema.graphql [hunk 10]
```diff
@@ -56,15 +88,26 @@ enum EventName {
...
   id: ID!
+  "network contract address"
```
Comment (MantisClone) [inline schema.graphql:93]: I think this should be "escrow contract address". Need to double check.

File: schema.graphql [hunk 11]
```diff
@@ -31,19 +49,33 @@ enum EscrowState {
 }
 
 type Escrow @entity {
+  "hash transaction of the escrow"
   id: ID!
+  "network contract address"
```
Comment (MantisClone) [inline schema.graphql:54]: I think this should be "escrow contract address" need to double check.

File: schema.graphql [hunk 12]
```diff
@@ -56,15 +88,26 @@ enum EventName {
...
 type EscrowEvent @entity {
+  "date payment set up"
   id: ID!
+  "network contract address"
   contractAddress: Bytes!
+  "is the reference data used to track the escrow event"
   reference: Bytes!
+  "smart contract that triggers when a payment is dispursed"
   escrow: Escrow!
+  "block event took place"
```
Comment (MantisClone) [inline schema.graphql:99]: Please change to "block in which event took place"

**Review by MantisClone — CHANGES_REQUESTED** [review]
I left a bunch of comments because I wanted to get early feedback. I need to double check some things because I'm not sure what the correct answer is.

**Comment by PaulieB14** [comment]
@MantisClone I made changes.  I'll be happy to revise any others.  Let me know

File: schema.graphql (hunk 7 above)
Comment (MantisClone) [reply schema.graphql:56]: I double-checked, and this should be "payment proxy contract address"

File: schema.graphql [hunk 13]
```diff
@@ -1,25 +1,43 @@
 type Payment @entity {
+  "hash transaction of the payment"
   id: ID!
+  "payment network contract address"
```
Comment (MantisClone) [inline schema.graphql:4]: Please change to "payment proxy contract address"

**Review by MantisClone — CHANGES_REQUESTED** [review]

File: schema.graphql [hunk 14]
```diff
@@ -56,15 +88,26 @@ enum EventName {
 }
 
 type EscrowEvent @entity {
+  "hash transaction of the escrow"
   id: ID!
+  "escrow contract address"
   contractAddress: Bytes!
+  "is the reference data used to track the escrow event"
   reference: Bytes!
+  "smart contract that triggers when a payment is disbursed"
```
Comment (MantisClone) [inline schema.graphql:97]: Please change to "the Escrow where this event occurred"

File: schema.graphql [hunk 15]
```diff
@@ -1,25 +1,43 @@
 type Payment @entity {
+  "hash transaction of the payment"
   id: ID!
+  "payment proxy contract address""
```
Comment (MantisClone) [inline schema.graphql:4]: Please remove extra `"`

**Review by MantisClone — CHANGES_REQUESTED** [review]

File: schema.graphql (hunk 7 above)
Comment (MantisClone) [reply schema.graphql:56]: @PaulieB14 Just pinging since Github hides this one. This one still needs to fixed too.

File: schema.graphql (hunk 14 above)
Comment (PaulieB14) [reply schema.graphql:97]: Corrected

File: schema.graphql (hunk 15 above)
Comment (PaulieB14) [reply schema.graphql:4]: corrected

File: schema.graphql (hunk 7 above)
Comment (PaulieB14) [reply schema.graphql]: Revised to corrected

**Pull request closed without merging**